docs/src/how-to-use.md

```diff
@@ -67,6 +67,8 @@ You can also check the [`test/inputs` folder](https://github.com/TulipaEnergy/Tu
 Below, we have a description of the files.
 At the end, in [Schemas](@ref schemas), we have the expected columns in these CSVs.
 
+> **Tip:** You can compare two CSV files with `git diff --no-index --word-diff-regex="[^[:space:],]+" file1 file2`.
```

Comment:
```suggestion
> **Tip:**
> If you modify CSV files and want to see your modifications, the normal `git diff` command will not be informative.
> Instead, you can use
>
> ```bash
> git diff --word-diff-regex="[^[:space:],]+"
> ```
>
> to make `git` treat the `,` as word separators.
> You can also compare two CSV files with
> 
> ```bash
> git diff --no-index --word-diff-regex="[^[:space:],]+" file1 file2
> ```
```
I added a line break in the "title" for consistency with the Notes and Warning, further explained the argument usage, and move the git commands to individual lines because the text became too long.

Reply:
What's the difference between the first bash and the second? In the first I don't see how you specify which files to compare. Also that command didn't work for me when I tried it (specifying the files like in bash 2) - so I added the --no-index argument to make it work.

Reply:
The `git diff` command by itself shows the current (not added/staged) changes in the repo. The `word-diff-regex` arguments tells it how to separate words.
The `--no-index` option tells the command to compare two files specifically, and that these files are not necessarily indexed by git. I didn't know about this option, actually. I normally only use the first one.

So, if you have two files to compare, you can use the second one. However, if you're just doing the normal `git diff` and that involves a CSV file, it might be beneficial to use the word diff regex.

To try:
- Open a CSV file, e.g., `test/inputs/Tiny/assets-data.csv`
- Change one single value
- In the terminal, run `git diff`. The row will be marked as modified
- (Optional) In the terminal, run `git diff --color-words` and `git diff --word-diff`. Neither will improve the situation.
- In the terminal, run `git diff --word-diff-regex="[^[:space:],]+"`. It will be more descriptive.
- (Optional) Choose a column and change all values for all rows and repeat the commands above.

Reply:
Ah okay. So it's when you're actively working in a CSV it shows you what you have changed. 
In that case I think these docs are fine - I already applied these changes so should be good!